### ISSUE TITLE
@samedson Could you help to check for this issue https://github.com/firebase/firebase-ios-sdk/issues/11471 ? Could this change make any fix for it?

### ISSUE THREAD

**Issue by pavm035** [firebase/firebase-ios-sdk#11854]
@samedson Could you help to check for this issue https://github.com/firebase/firebase-ios-sdk/issues/11471 ? Could this change make any fix for it?

_Originally posted by @bestwnh in https://github.com/firebase/firebase-ios-sdk/issues/11463#issuecomment-1605849509_
            

Looks like archive is failing due to `${BUILD_DIR}/../../` not resolving properly

Error:
Unable to load contents of file list: '<${BUILD_DIR}/../../>/SourcePackages/checkouts/firebase-ios-sdk/Crashlytics/CrashlyticsInputFiles.xcfilelist'

**Comment by paulb777** [comment]
Please identify the Xcode and Firebase versions. Note, it should be addressed in the latest versions.

**Comment by pavm035** [comment]
@paulb777 
Xcode 15.0(15A240d)
Firebase: 10.13.0

**Comment by paulb777** [comment]
Thanks. Please fill out the full issue template since #11463 was resolved.

**Comment by pavm035** [comment]
@paulb777 

### Description
When i use Xcode 15 to archive with mentioned steps aboe in #11463  I'm getting below error

Unable to load contents of file list: '<${BUILD_DIR}/../../>/SourcePackages/checkouts/firebase-ios-sdk/Crashlytics/CrashlyticsInputFiles.xcfilelist'

### Reproducing the issue
Always

### Firebase SDK Version
10.13.0

### Xcode Version
15.0 RC

### Installation Method
Swift Package Manager

### Firebase Product(s)
Crashlytics

### Targeted Platforms
iOS


### Package.resolved
```swift
{
   {
      "identity" : "firebase-ios-sdk",
      "kind" : "remoteSourceControl",
      "location" : "https://github.com/firebase/firebase-ios-sdk.git",
      "state" : {
        "revision" : "df2171b0c6afb9e9d4f7e07669d558c510b9f6be",
        "version" : "10.13.0"
      }
}
````

**Comment by pavm035** [comment]
@paulb777 

### Description
When i use Xcode 15 to archive with mentioned steps aboe in #11463  I'm getting below error

Unable to load contents of file list: '<${BUILD_DIR}/../../>/SourcePackages/checkouts/firebase-ios-sdk/Crashlytics/CrashlyticsInputFiles.xcfilelist'

### Reproducing the issue
Always

### Firebase SDK Version
10.13.0

### Xcode Version
15.0 RC

### Installation Method
Swift Package Manager

### Firebase Product(s)
Crashlytics

### Targeted Platforms
iOS


### Package.resolved
```swift
{
   {
      "identity" : "firebase-ios-sdk",
      "kind" : "remoteSourceControl",
      "location" : "https://github.com/firebase/firebase-ios-sdk.git",
      "state" : {
        "revision" : "df2171b0c6afb9e9d4f7e07669d558c510b9f6be",
        "version" : "10.13.0"
      }
}
````

**Comment by themiswang** [comment]
Close this issue, similar to issue https://github.com/firebase/firebase-ios-sdk/issues/11836.

The directory for archive build is a bit different from the general debug/release build, this is probably the correct path `${BUILD_DIR}/../../../../../SourcePackages/checkouts/firebase-ios-sdk/Crashlytics/CrashlyticsInputFiles.xcfilelist` you need to specify source packages for an archive build using environment config `BUILD_DIR` as a relative path. An easier solution might be copy this xcflielist content to your local project so you can import the file list directly from your project absolute path.

**Comment by pavm035** [comment]
@themiswang  yes i tried to switch to parent directory using this pattern `${BUILD_DIR}/../../../../../` but it didn't resolve to switching back to nested parent directory, so not sure if it's an xcode issue.
